torch/_inductor/select_algorithm.py

```diff
@@ -181,6 +184,26 @@ def stride(self, name, index):
             val = self.named_input_nodes[name].get_stride()[index]
         return texpr(self.rename_indexing(val))
 
+    def modification(self, fixed_inputs):
+        # HACK, but I can't figure out how to reuse existing Triton codegen properly
+        class PlaceholderSubstitution(V.MockHandler):
+            self.name = "PlaceholderSubstitution"
+
+            def load(self, name: str, index: sympy.Expr):
+                return f"({fixed_inputs[name]})"
+
+            def constant(self, value, dtype):
+                return TritonOverrides.constant(value, dtype)
+
+            def __getattr__(self, name):
+                return getattr(TritonOverrides, name)
+
+
+        with TritonKernel(1, 1, index_dtype=torch.int64) as kernel:
+            with V.set_ops_handler(PlaceholderSubstitution()):
```

Comment:
You will want to wrap the existing V.ops if you want code output to work properly.  With this version I think you will just get one giant string with no newlines.